homepage/homepage/content/docs/groups/intro.mdx

```diff
@@ -11,6 +12,16 @@ Every CoValue has an owner, which can be a `Group` or an `Account`.
 You can use a `Group` to grant access to a CoValue to **multiple users**. These users can
 have different roles, such as "writer", "reader" or "admin".
 
+CoValues owned by an Account can only be accessed by that Account. Additional collaborators cannot be added,
+and the ownership cannot be transferred to another Account. This makes account ownership very rigid.
+
+Creating a Group for every new CoValue is a best practice, even if the Group only has a single user in it
+(this is the default behavior when creating a CoValue with no explicit owner).
+
+<Alert variant="info" className="flex gap-2 items-center my-4">
+  Account ownership is kept for backwards compatibility, but will be removed in a future release.
+</Alert>
```

Comment:
I personally would make it even more discouraging, WDYT?

```suggestion
<Alert variant="warning" className="flex gap-2 items-center my-4">
  While creating CoValues with Accounts as owners is still technically possible for backwards compatibility, it will be removed in a future release.
</Alert>
```